### ISSUE TITLE
Link buttons do not have draggable=false

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk_publishing_components#1558]
Was just looking at this for something else at the FT and noticed GOV.UK Frontend has 'draggable=false' set on link buttons but this has not been ported to here.

I think this'll be a quick fix, I hope:

See original pull request that introduced it: https://github.com/alphagov/govuk-frontend/pull/1020#issue-219981238

Should be able to check if the element is a link and set draggable false here: https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/button_helper.rb

And then there'd need to be a small test added.
